packages/design-system-scripts/package.json

```diff
@@ -32,9 +32,13 @@
     "enzyme": "3.10.0",
     "enzyme-adapter-react-16": "1.14.0",
     "enzyme-to-json": "3.3.5",
-    "eslint": "^4.18.1",
+    "eslint": "^6.8.0",
```

Comment:
@pwolfert Should `eslint`, `stylelint`, `prettier` be peer deps? I believe they are also needed in the root package.json's `husky` precommit hooks. I originally tried moving these to peer deps and installing them to the root package.json, but [it didn't work easily](https://github.com/CMSgov/design-system/pull/672/commits/ef73463922e4acc6c0f445f18615a16eaebd6d65), maybe because the `design-systems-scripts` package is not located with the `node_modules` at the root.

Reply:
I think they should probably be normal `dependencies` because the user (child design system developer) expects `cmsds lint` to just work. In the case of the core DS, we might need it for other things, but we don't force the child design system to use any linter if they don't want to (and we don't require them to delint before committing). That's my opinion at least

Reply:
gotcha that makes sense! so if we dont want to include `husky` precommit hooks, do you think i should add `prettier` to the linter too? Right now our eslint and stylelint are configured to work with `prettier`

Reply:
I decided to include prettier in the lint scripts.

On a related topic though, @pwolfert how do you think we should handle `react` dependencies in the packages? `react` is a peer dep in `design-system`, but a normal dep in `design-system-docs` and `design-system-scripts`.

Reply:
I think it should still be a peer dep for dependents (users) since we want them to be in control of which version of react they use, but for the child design systems, we it's probably okay to prescribe the version by making it a real dependency...or we could also leave it up to child design systems to decide, and then it's always a peer dependency and we make react a real dependency of the root of our design system.

Reply:
im ok with leaving it to the child DS to decide, and just make react and react-dom a peer dep across all the packages

Reply:
Because of this PR: https://github.com/CMSgov/design-system/pull/721,
I think it makes sense to make eslint, stylelint and prettier peer deps